### DISCUSSION TITLE
Tidal support any time soon?

### DISCUSSION THREAD

**Discussion by fischett** [ShankarBUS/ModernFlyouts#1375] [Ideas]
Hi Everyone,

Overall, amazing app. Have been using it for a couple of years already!

Wanted to ask or suggest, because I found nothing in the discussion about supporting Tidal. Will this be under consideration in the future? As it is a streaming service for Hi-Res audio, I thought some people would be interested in.

It partially works if one sets Windows 10 as the source. But I found that it does not update the progress bar when a song is playing. Also, the play/pause button works with a bit of a delay, a couple of seconds (pause works well, but it takes like 5 sec to let you continue playing the song).

Thanks!

**Comment by prakharb5** [comment]
The app doesn't introduce support for apps, it just shows apps which already support Windows' APIs for media sessions (GSMTC). Ideally, you can request Tidal devs to add support for this.
